flowey/flowey_core/src/pipeline.rs

```diff
@@ -250,6 +252,12 @@ impl Default for GhPrTriggers {
         Self {
             branches: Vec::new(),
             exclude_branches: Vec::new(),
+            types: vec![
```

Comment:
I'm of two minds with this default.

On one hand, this seems like a reasonable default, as it lines up with the behavior folks new to GitHub actions might expect to be the case.

On the other hand, this is not actually true to the default semantics of the underlying GitHub actions platform...

Maybe we can split the difference, and remove the existing `impl Default`, swapping it out with a standalone `pub fn new_draftable()` method (name subject to bikeshedding), which more clearly documents the "sane defaults" it sets?

I think having the `impl Default` do "non trivial" defaults is a recipe for pain, as there's no obvious place to document what defaults were chosen that a user would obviously stumble upon, whereas having a standlone method with docs makes that info a lot easier document.